### PR TITLE
Added a cmd to get a course user's information.

### PR DESCRIPTION
**Pull request by OliverLok**

### CODE REVIEW

File: cmd/course-user-get/main_test.go [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+			targetEmail: "course-student@test.edulinq.org",
+			courseID:    "course101",
+		},
+		{
+			CommonCMDTestCase: cmd.CommonCMDTestCase{
+				ExpectedStdout: USER_NOT_FOUND,
+			},
+			targetEmail: "ZZZ",
+			courseID:    "course101",
+		},
```
Comment (Lucas-Ellenberger) [inline cmd/course-user-get/main_test.go:34]: Add one more test case for a user that exists in the database, but is not enrolled in the course.

See the python interface test case for reference (https://github.com/edulinq/autograder-py/blob/main/tests/api/testdata/courses/users/courses_users_get_not_enrolled.json).

**Review by Lucas-Ellenberger — CHANGES_REQUESTED** [review]
Looking good!

Just asking for one more test case.

**Review by Lucas-Ellenberger — APPROVED** [review]
lgtm!

**Review by eriq-augustine — APPROVED** [review]
lgtm

**Pull request merged**